src/components/nimbleSelect/NimbleSelect.tsx

```diff
@@ -92,7 +92,7 @@ export const NimbleSelect: React.FC<NimbleSelectProps> = ({
   const handleChnage = (event: any) => {
     const val = event.target.value;
     multiple ? setSelectedValueForMultiple(val.filter((item: string) => item !== '-')) : setSelectedValue(val);
-    onChange && onChange(val);
```

Comment:
in initial select `val` contains the default value, so it looks like` ['-',2]`
in order to correct this I am filtering the` '-'`